tests/nn/test_inference.py

```diff
@@ -1381,3 +1386,42 @@ def test_flow_tracker(centered_pair_predictions: Labels, tmpdir):
         for key in tracker.candidate_maker.shifted_instances.keys():
             assert lf.frame_idx - key[0] <= track_window  # Keys are pruned
             assert abs(key[0] - key[1]) <= track_window  # References within window
+
+
+def test_movenet_inference():
+    inference_layer = MoveNetInferenceLayer(model_name="lightning")
+    inference_model = MoveNetInferenceModel(inference_layer)
+
+    video = sleap.Video.from_image_filenames(["tests/data/videos/white_guy.png"])
+
+    p = sleap.pipelines.Pipeline(sleap.pipelines.VideoReader(video))
```

Comment:
Can we just re-use the movenet.mp4 here and remove the white_guy.png
```suggestion
    # TODO (Jiaying): Add a fixture named `movenet_video` that returns a `Video` object created from dance.mp4
    video = movenet_video
    p = sleap.pipelines.Pipeline(sleap.pipelines.VideoReader(video, example_indices=[0]))
```